src/obfl-specification.html

```diff
@@ -1565,6 +1566,8 @@ <h4 id="L1105" class="include">toc-entry</h4>
 
 <p>The toc-entry element defines an entry in a table of contents.</p>
 
+<ins datetime="2019114T14:00:00">The contents of a toc-entry should be put in a child toc-entry-content element instead of in the toc-entry element itself.</ins>
```

Comment:
I wouldn't say "instead of in the toc-entry element itself". It's not because this used to be supported in older versions that we need to mention it in the latest version. It's clear from the sections below that only `toc-entry-content` are allowed inside `toc-entry`.